source/modules/administration/forms/forms.rst

```diff
@@ -0,0 +1,889 @@
+
+.. warning:: Form migration must be done from the GLPI 10 database.
+
+Once you have completed the migration from GLPI 10 to GLPI 11, simply enter the command from your GLPI folder:
+
+``php bin/console migration:formcreator_plugin_to_core``
+
+Forms by default
+----------------
+
+There are two pre-created forms by default:
+
+* Report an issue
+* Request a service
+
+.. image:: images/forms_view_self-service.png
+   :alt: view home self-service profile
+   :scale: 37%
+
+These 2 forms are accessible to everyone. They can be deactivated/deleted/modified
+
+
+List of forms
+-------------
+
+The list of forms is available from **Administration** > **Forms**.
+They will also be visible from **Assistance** > **Service catalog**. This tab refers to the self-service profile home page (see :doc:`Service catalog</modules/assistance/service-catalog>`).
+
+
+.. image:: images/forms_list.png
+   :alt: List of forms
+   :scale: 42%
+
+Forms options
+-------------
+
+When creating a form (by **+ Add**), several options will be available to customize and facilitate the user experience as much as possible.
+
+.. image:: images/add_form.png
+   :alt: Add a new form
+   :scale: 42%
+
+Customize the formatting
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tools are available to customize the organization of your forms
+
+.. image:: images/custom_formatting.png
+   :alt: Customize the formatting
+   :scale: 100%
+
+#. Add a new question
+#. Add a comment
+#. Add a new section
+#. Add an horizontal layout
+
+
+You can also move your questions to reorganize your form as you wish
+
+.. image:: images/move_question.png
+   :alt: Reorganize your forms
+   :scale: 80%
+
+You also have tools to reorganize an entire section
+
+.. image:: images/section_option.png
+   :alt: Option for sections
+   :scale: 69%
+
+
+Basic informations
+~~~~~~~~~~~~~~~~~~
+
+* Modify the form's title in the **Underlined form** file (this title will be visible to people with access to the form)
+* Add a **description** to simplify the use of the form (this description will be visible to people with access to the form)
+
+Add questions
+~~~~~~~~~~~~~
+
+Each question has its own **title** field (identified by new question) and will be **visible to the user**.
+Each question **can be made mandatory** so that the user has no other choice than to fill in this field to validate their form
+
+There are actually 11 possible question types:
+
+* **Short answer**: Text field with limited character size. This field has no formatting (bold, italics, color, etc.).
+
+.. image:: images/short_answer.png
+   :alt: Short answer option
+   :scale: 80%
+
+This field has additional options:
+
+* *Text*: Allows you to enter text (alphanumeric)
+* *Emails*: Allows you to enter email addresses only (xxxxxx@xxxxxx.xx)
+* *Number*: Allows you to enter numeric characters only
+
+
+* **Long answer**: unlimited text field in characters. The formatting of this field can be customized by the user (bold, italic, color, etc.)

```

Comment:
Isn't there a DB column limit of 65,535 characters for all combined answers in a JSON format?